src/core/constants.ts

```diff
@@ -0,0 +1,12 @@
+export const STATIC_ARRAY_REGEX = /^(.+)\[(\d+)\]$/
+export const UFIXED_REGEX = /^ufixed(\d+)x(\d+)$/
+
+export const MALFORMED_ADDRESS_ERROR_MSG = 'address seems to be malformed'
```

Comment:
We'll have some conflicts on this file. I've generally opted to remove `ALGORAND_` prefix, as it feels a little redundant.